### PR TITLE
Fix TOC issue

### PR DESCRIPTION
**Pull request by vikram-redhat**
This PR:

* Adds a single blank line before toc::[] in all assemblies.
* Updates guidelines to make sure the assembly metadata specifies that a blank line be added after initial document header information and a toc.
* Fixes the common-attributes.adoc to remove unnecessary attributes.

A separate PR will need to be created for each release branch.

### CODE REVIEW

File: applications/deployments/deployment-strategies.adoc [hunk 1]
```diff
@@ -3,6 +3,7 @@
 include::modules/common-attributes.adoc[]
 :context: deployment-strategies
 
+
```
Comment (pneedle-rh) [inline applications/deployments/deployment-strategies.adoc:6]: Some of the assemblies now have two preceding blank lines. Is it worth only adding blank lines to the assemblies that do not current have one between the header and the `toc` declaration, for consistency?

Comment (abrennan89) [reply applications/deployments/deployment-strategies.adoc:6]: Yep, I've noticed this for the couple of serverless ones I've looked at so far also

Comment (pneedle-rh) [reply applications/deployments/deployment-strategies.adoc:6]: I am trying to script a solution for this.

**Review by JStickler — APPROVED** [review]
ACK from service mesh team.  Just let us know when it merges so we make sure we rebase our PRs (we don't want to overwrite this accidentally!).

**Comment by ahardin-rh** [comment]
LGTM!

File: applications/projects/creating-project-other-user.adoc [hunk 2]
```diff
@@ -2,10 +2,11 @@
...
+
 toc::[]
 
 Impersonation allows you to create a project as a different user.
 
 include::modules/authentication-api-impersonation.adoc[leveloffset=+1]
 
-include::modules/impersonation-project-creation.adoc[leveloffset=+1]
\ No newline at end of file
+include::modules/impersonation-project-creation.adoc[leveloffset=+1]
```
Comment (pneedle-rh) [inline applications/projects/creating-project-other-user.adoc:12]: Where you see the last line in a file being removed and added in this PR, this is the script that I ran adding new lines to the end of a file where one did not previously exist. It was a side effect of the script, but it is useful.

File: applications/deployments/deployment-strategies.adoc (hunk 1 above)
Comment (pneedle-rh) [reply applications/deployments/deployment-strategies.adoc]: I ran the script against a local clone of this PR and then pushed the changes to the original commit. There is now one blank line preceding the `toc::[]` line in each file.

Comment (vikram-redhat) [reply applications/deployments/deployment-strategies.adoc]: Thank you @pneedle-rh

**Comment by tradej** [comment]
LGTM.

**Comment by Preeticp** [comment]
Verified for odo, ODC, Helm, Build, Pipelines, GitOps, Logging. LGTM. Thank you @pneedle-rh and @vikram-redhat

**Comment by vikram-redhat** [comment]
@pneedle-rh I am going to merge this. Please merge the rest of the release branch PRs.

**Pull request merged**